.buildkite/commands/checkout-release-branch.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash -eu
 
-# BUILDKITE_RELEASE_VERSION is passed as an environment variable from fastlane to Buildkite
+# Note: BUILDKITE_RELEASE_VERSION is passed as an environment variable from fastlane to Buildkite
```

Comment:
Isn't it "from Buildkite to fastlane" instead? Actually, I'm not sure how fastlane is involved in this. The env var is passed from Rv2 to Buildkite which is then calling this `.sh` which uses it… no fastlane involved in there?

Reply:
Yeah, that was the original comment and I agree it's a bit confusing. Tried to improve it a bit on 0dd6da9595f03fd900d8584c345ed388656af5ba.